sos/report/plugins/powerpc.py

```diff
@@ -74,7 +77,10 @@ def setup(self):
                 "serv_config -l",
                 "bootlist -m both -r",
                 "lparstat -i",
-                "ctsnap -xrunrpttr -d %s" % (ctsnap_path)
+                "ctsnap -xrunrpttr -d %s" % (ctsnap_path),
+                "lsdevinfo",
+                "journalctl",
```

Comment:
Didn't see this earlier, but nack to this blind `journalctl` call. We get the system journal in the `logs` plugin already, and if you want/need a service specific journal you can use `Plugin.add_journal()`.

Reply:
`journalctl --no-pager` is the full content of the journal, it just pushes the entire content to stdout without paging - it doesn't truncate the output in any way.

Reply:
Any truncating you're seeing is from the size limiting in place, not the command being run. Journals are limited by default to the latest 100MB. If you truly need the entire journal regardless of size then use `--log-size=0` in your sos command.

In any event, the `logs` plugin already does sufficient journal collection for non-unit-specific collections, and the `powerpc`  plugin would not be the place for such a collection anyways.

Reply:
ok ..Thanks for the info
I think we have all data collected in following path
$ls sos_commands/logs
journalctl_--disk-usage  journalctl_--no-pager  journalctl_--no-pager_--catalog_--boot  journalctl_--no-pager_--catalog_--boot_-1  ls_-alRh_.var.log